tests/E2E/test/campaigns/common_scenarios/category.js

```diff
@@ -48,9 +48,10 @@ module.exports = {
       test('should click on "Save" button', () => {
         return promise
           .then(() => client.scrollWaitForExistAndClick(CategorySubMenu.save_button, 50))
-          .then(() => client.getTextInVar(CategorySubMenu.category_number_span, "number_category"));
+          .then(() => client.getTextInVar(CategorySubMenu.category_number_span, "number_category"))
+          .then(() => tab['number_category'] = tab['number_category'].split('(')[1].split(')')[0]);
```

Comment:
Not the best way to select this item but I guess you had nothing easier to use.

For future puppeeter tests, when you have such complex usecases, let's work together in order to to provide you easier items to use 😉